src/multibody/model.hpp

```diff
@@ -100,9 +100,13 @@ namespace se3
     std::vector<Frame> frames;
     
     /// \brief Vector of subtrees.
-    /// subtree[j] corresponds to the subtree supported by the joint j.
+    /// subtrees[j] corresponds to the subtree supported by the joint j.
```

Comment:
For the support, is only available for backward pass, not forward! And lastChild is in Data, not in Model. So, algo without data will not work !

So I suggest for simplicity and efficiency to keep supports and subtrees.